code/modules/surgery/organs/augments_internal.dm

```diff
@@ -83,7 +86,7 @@
 
 /obj/item/organ/cyberimp/brain/anti_drop/proc/release_items()
 	for(var/obj/item/I in stored_items)
-		I.flags ^= NODROP
```

Comment:
guess i could've made both activate/deactivate use the release_items proc(and rename it)